### PR TITLE
fix: add propagation time to test/setup

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

**Comment by apeabody** [comment]
I believe the CI should be unblocked #717

File: test/setup/main.tf [hunk 1]
```diff
@@ -49,3 +49,10 @@ module "project" {
     "billingbudgets.googleapis.com",
   ]
 }
+
+# propagation time
+resource "time_sleep" "wait_180_seconds" {
```
Comment (daniel-cit) [inline test/setup/main.tf:54]: the integration build has a step to wait for propagation (services, iam) from the setup

https://github.com/terraform-google-modules/terraform-example-foundation/blob/3e2ca413d2fd57c309748109f56008be2503ff9a/build/int.cloudbuild.yaml#L25-L27

maybe the sleep time in the step in `int.cloudbuild.yaml` could be changed if it is not enough anymore

Comment (daniel-cit) [reply test/setup/main.tf:54]: looking at some of the build errors with `SERVICE_DISABLED`, it looks like that the APIs not enabled errors are in the `seed` and `cicd` projects that are created by the bootstrap module. 
Maybe this change could be implemented there or in the project-factory module

Comment (daniel-cit) [reply test/setup/main.tf:54]: looking at the error messages, the project that is been reported as not having the APIs enable is **the project created in the setup step**. 
Adding the extra minutes to the [int.cloudbuild.yaml](https://github.com/terraform-google-modules/terraform-example-foundation/blob/3e2ca413d2fd57c309748109f56008be2503ff9a/build/int.cloudbuild.yaml#L25-L27) file should reduce the problem

Comment (apeabody) [reply test/setup/main.tf:54]: Hi @daniel-cit thanks for the feedback and suggestions!  I'll review it sometime next week.  Cheers!

Comment (apeabody) [reply test/setup/main.tf:54]: Thanks @daniel-cit - I see https://github.com/terraform-google-modules/terraform-example-foundation/blob/90f15f28c03f391dc95aaa843f1a5727772bd8e4/build/int.cloudbuild.yaml#L27 was committed last week, so I'll close out this PR.

**Comment by apeabody** [comment]
Close in favor of https://github.com/terraform-google-modules/terraform-example-foundation/blob/90f15f28c03f391dc95aaa843f1a5727772bd8e4/build/int.cloudbuild.yaml#L27

**Pull request closed without merging**